pbjson-build/src/generator/message.rs

```diff
@@ -380,6 +380,11 @@ fn write_serialize_scalar_variable<W: Write>(
                 "{}#[allow(clippy::needless_borrow)]",
                 Indent(indent)
             )?;
+            writeln!(
+                writer,
+                "{}#[allow(clippy::needless_borrows_for_generic_args)]",
```

Comment:
apparently the new version of clippy flags some of this code as "needless borrows" but under a different lint (`needless_borrow` previously and now `needless_borrows_for_generic_args`)